CI-CD/CodeFresh/CodeFresh-NPM

```diff
@@ -0,0 +1,24 @@
+# More examples of Codefresh YAML can be found at
+# https://codefresh.io/docs/docs/yaml-examples/examples/
+
+version: "1.0"
+stages:
+  - "WhiteSource"
+
+steps:
+  Freestyle:
+    title: WhiteSource-E2E
+    type: freestyle
+    stage: "WhiteSource"
+    working_directory: "${{CF_BRANCH}}"
+    arguments:
+      image: 'quay.io/codefreshplugins/alpine:3.8'
+      commands:
+        # Add WS_APIKEY & WS_WSS_URL as environment variables 
+        - apk update
+        - apk add openjdk8 curl npm git
+        - git clone https://github.com/some/repo
+        - cd repo
+        - curl -LJO https://unified-agent.s3.amazonaws.com/wss-unified-agent.jar
+        - npm install
+        - java -jar wss-unified-agent.jar -product CodeFresh -project $CF_BRANCH -logLevel debug
```

Comment:
Use env variables instead of command line parameters

Reply:
added